README.md

```diff
@@ -33,61 +34,45 @@ The common component that implements the 3GPP TS v27.007 standard has been writt
-
-Please refer to the README of [Lab-Project-FreeRTOS-Cellular-Demo](https://github.com/FreeRTOS/Lab-Project-FreeRTOS-Cellular-Demo) project for more information.
+* docs : documentations
 
-## Integrate FreeRTOS Cellular Library with MCU platforms
+## Integrate FreeRTOS Cellular Interface with MCU platforms
 
-The FreeRTOS Cellular Library runs on MCUs.  It use an abstracted interface - the [Comm Interface](https://github.com/FreeRTOS/Lab-Project-FreeRTOS-Cellular-Library/blob/master/include/cellular_comm_interface.h), to communicate with cellular modems.  A Comm Interface must be implemented as well on the MCU platform.  The most common implementations of the Comm Interface are over UART hardware, but it can be implemented over other physical interfaces such as SPI as well.   The documentation of the Comm Interface is found within the [Cellular API References](https://www.freertos.org/Documentation/api-ref/cellular/comm_if.html). These are example implementations of the Comm Interface:
+The FreeRTOS Cellular Interface runs on MCUs.  It uses an abstracted interface - the [Comm Interface](https://github.com/FreeRTOS/FreeRTOS-Cellular-Interface/tree/main/source/interface/cellular_comm_interface.h), to communicate with cellular modems. A Comm Interface must be implemented as well on the MCU platform.  The most common implementations of the Comm Interface are over UART hardware, but it can be implemented over other physical interfaces such as SPI as well. The documentation of the Comm Interface is found within the [Cellular API References](https://www.freertos.org/Documentation/api-ref/cellular/comm_if.html). These are example implementations of the Comm Interface:
 
-* [FreeRTOS windows simulator comm interface](https://github.com/FreeRTOS/Lab-Project-FreeRTOS-Cellular-Demo/blob/master/source/cellular/comm_if_windows.c)
+* FreeRTOS windows simulator comm interface
 * [FreeRTOS Common IO UART comm interface](https://github.com/aws/amazon-freertos/blob/feature/cellular/vendors/st/boards/stm32l475_discovery/ports/comm_if/comm_if_uart.c)
 * [STM32 L475 discovery board comm interface](https://github.com/aws/amazon-freertos/blob/feature/cellular/vendors/st/boards/stm32l475_discovery/ports/comm_if/comm_if_st.c)
 * [Sierra Sensor Hub board comm interface](https://github.com/aws/amazon-freertos/blob/feature/cellular/vendors/sierra/boards/sensorhub/ports/comm_if/comm_if_sierra.c)
 
-The FreeRTOS Cellular Library uses kernel APIs for task synchronization and memory management.  Please refer to "cellular/cellular_platform.html" after unzipping the doc to see the needed API.
```

Comment:
Do we have cellular_platform.h and cellular_platform.c file for user since this part is removed?

Reply:
This repo doesn't have cellular_platform.h and cellylar_platform.c, they will exist in demo application. The only exception is there is a cellular_platform.h in unit test, but this one is for testing.

Reply:
This part can point to the demo application after the demo application added into the prod 3.